src/components/Converter.jsx

```diff
@@ -178,6 +179,11 @@ class Converter extends Component {
           status: DONE,
           fileName: this.state.operation === MERGE ? fileName : `File${filtered.length > 1 ? 's' : ''}`
         })
+
+        if (document.hidden) {
+          notify({ didSucceed: true, operation: this.state.operation})
```

Comment:
Not needed for this PR, but do we want to make a more generic `notify` API for potential future notifications?

Reply:
Yeah, we definitely should if more use cases arise!